src/renderer/components/ft-icon-button/ft-icon-button.js

```diff
@@ -114,7 +126,19 @@ export default Vue.extend({
       }
     },
 
-    handleDropdownClick: function ({ url, index }) {
+    handleDropdownClick: function ({ url, index }, event) {
+      if (!this.$handleDropdownKeyboardEvent(event, event?.target)) {
+        return
+      }
+
+      // const listbox = $(`#${this.id}`)
+      // const allOptions = listbox.children()
+
+      // allOptions.attr('aria-selected', 'false')
+      // allOptions.attr('tabindex', '-1')
+      // event.target.setAttribute('aria-selected', 'true')
+      // event.target.setAttribute('tabindex', '0')
```

Comment:
What happen to this commented code?

Reply:
@jasonhenriquez

Reply:
@jasonhenriquez would you be able to provide context on the removal? maybe someone else could work on it. (I think this is the main blocker for the code to be reviewed)